types/jquery/README.md

```diff
@@ -0,0 +1,53 @@
+### Usage
+
+#### Global
+
+When jQuery is globally available, you can use `jQuery` and `$` directly.
+
+#### Importing (with a global DOM available)
+
+When you want to import jQuery as a module and have a global DOM available (e.g. browser and browser-like environments):
+
+```typescript
+import * as jQuery from 'jquery';
```

Comment:
It will type check, but the emitted code will be wrong unless you are using --modules es2015